### ISSUE TITLE
useLayoutEffect is not defined error

### ISSUE THREAD

**Issue by asterikx** [bvaughn/react-resizable-panels#118]
After upgrading to v0.0.38, I get this error with Next.js:

```
Unhandled Runtime Error
ReferenceError: $jhddX$useLayoutEffect is not defined

Call Stack
eval
../node_modules/next-translate-plugin/lib/cjs/node_modules/react-resizable-panels/dist/react-resizable-panels.module.js (6:0)
../../node_modules/react-resizable-panels/dist/react-resizable-panels.module.js
```

v0.0.37 works just fine.

**Comment by YMNNs** [comment]
I have the same issue but v0.0.37 works.

**Comment by NMinhNguyen** [comment]
Can confirm this breaks in a fresh Vite app as well (`yarn create vite`, `yarn add react-resizable-panels`, import something from the package and `yarn run dev` to start it). I believe it's due to the `toString()` change: https://github.com/bvaughn/react-resizable-panels/commit/ed9694750595104727fa5f8d0c58b3d1c386ece7

**Comment by bvaughn** [comment]
> I believe it's due to the `'useId'.toString()` change: https://github.com/bvaughn/react-resizable-panels/commit/ed9694750595104727fa5f8d0c58b3d1c386ece7

That would also be my first guess.

**Comment by NMinhNguyen** [comment]
> > I believe it's due to the `'useId'.toString()` change: https://github.com/bvaughn/react-resizable-panels/commit/ed9694750595104727fa5f8d0c58b3d1c386ece7
> 
> 
> 
> That would also be my first guess.

Yeah, I checked out the repo and built the package a few times, and the `'useId'.toString()` seems to be confusing Parcel. It seems like switching to a namespace import of React everywhere works though.

**Comment by bvaughn** [comment]
I'm going to publish 0.0.39 with a fix for this in a few. I'm looking at other options for a few first though (beyond just reverting it).

**Comment by bvaughn** [comment]
> It seems like switching to a namespace import of React everywhere works though.

Thanks for this tip 👍🏼  I've confirmed it locally. Now I'm just adding some lint rules and stuff to make sure I don't regress.

**Comment by bvaughn** [comment]
Should be fixed in 0.0.39
